code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -169,6 +182,42 @@
 	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 
+/obj/item/weapon/nullrod/scythe/talking
+	icon_state = "talking_sword"
+	item_state = "talking_sword"
+	name = "possessed blade"
+	desc = "When the station falls into chaos, it's nice to have a friend by your side."
+	attack_verb = list("chopped", "sliced", "cut")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	var/possessed = FALSE
+
+/obj/item/weapon/nullrod/scythe/talking/attack_self(mob/living/user)
+	if(possessed)
+		return
+	if(user.mind && (user.mind.assigned_role != "Chaplain"))
+		return
+
+	user << "You attempt to wake the spirit of the blade..."
+
+	possessed = TRUE
+
+	var/list/mob/dead/observer/candidates = pollCandidates("Do you want to play as the spirit of [user.real_name]'s blade?", ROLE_PAI, null, FALSE, 100)
+	var/mob/dead/observer/theghost = null
+
+	if(candidates.len)
+		theghost = pick(candidates)
+		var/mob/living/simple_animal/shade/S = new(src)
+		S.real_name = name
+		S.ckey = theghost.ckey
+		var/input = stripped_input(user,"What do you want to name your blade?", ,"", MAX_NAME_LEN)
```

Comment:
I think the ghost should be able to name it. It'd be fun to spring a terrifying evil sword on someone.